src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.site-navigation-list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
```

Comment:
there is no need for paddings here

Reply:
I was try to remove paddings, but when i remove him, i have 2 mistakes in test, so i save him